app/assets/javascripts/govuk-component/tasklist.js

```diff
@@ -64,54 +70,61 @@
       // the case where the user has expanded all steps, so they are not returned to a particular step, but
       // still could have scrolled a long way down the page.
       function storeScrollPosition() {
-        closeAllSteps();
+        hideAllSteps();
         var $step = getStepForAnchor();
 
         document.body.scrollTop = $step && $step.length
           ? $step.offset().top
           : 0;
       }
 
-      function addOpenCloseAllButton() {
-        $element.prepend('<div class="pub-c-task-list__controls"><button aria-expanded="false" class="pub-c-task-list__button pub-c-task-list__button--controls js-step-controls-button">' + bulkActions.openAll.buttonText + '</button></div>');
+      function addShowHideAllButton() {
+        $element.prepend('<div class="pub-c-task-list__controls"><button aria-expanded="false" class="pub-c-task-list__button pub-c-task-list__button--controls js-step-controls-button">' + bulkActions.showAll.buttonText + '</button></div>');
```

Comment:
I wonder if this button has enough context for screenreader users. It's the first thing they will land on when reaching the tasklist, and it's not really clear what content they will expand by pressing it. Maybe it needs some kind of aria-label to indicate what it relates to?

Reply:
Yeah, that was something we discussed during one the accessibility reviews for this. Apparently it's fine. It does have an aria-controls attribute appended later as well, which adds a bit more context.